### DISCUSSION TITLE
Pro version in a dual boot with Linux configuration?

### DISCUSSION THREAD

**Discussion by mrjoecastellon** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#790] [Q&A] [answered]
I have a 2015 base 13" MacBook Pro. I've been using AlDente free with great success. 

Since this is a very old MacBook, I want to install Linux to revitalize it. My intention is to have a dual boot setup so that I can have AlDente installed.

I understand that while booted on Linux, AlDente has no effect because the limit is not set at the hardware level. In fact, I have already tested this and see that my MacBook will continue to charge past the set limit (75%) whenever I'm running Linux. 

So my question is, if I buy the Pro version of AlDente and set the charge limit to 75%, will it continue to have this effect while running Linux?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @mrjoecastellon,

Due to the different technical pathways used to limit charging for Intel MacBooks without Thunderbolt 3, the charge limit should persist even if you boot into Linux, at least it worked for me in Windows on my old MacBook. However, the final charge limit in Linux might be 5-10% off.

Please try it again:
1. Set the Charge limit to 80% in macOS
2. Boot into Linux and check if charging stops between 80-90%

Let me know if it worked.

Cheers,
Matthias

**Reply by mrjoecastellon** [reply]
Thank you for your quick reply! 

Just to be clear, this will only work with the pro version of AlDente, correct?

**Reply by MatthiasKerbl** [reply]
Yes, this works only in AlDente Pro. Should have mentioned it above.
You can read more about that in this blog post: https://apphousekitchen.com/feature-explanation-stop-charging-when-powered-off-app-closed/
The 3. paragraph is the important one for your MacBook model.

**Reply by mrjoecastellon** [reply]
Awesome! Thank you so much!

**Reply by mrjoecastellon** [reply]
Just wanted to add, I bought AlDente Pro and set the charging limit to 75%. I booted into Linux Endeavour and the battery has successfully remained at 75%.

AlDente Pro is worth every damn penny!

**Reply by MatthiasKerbl** [reply]
I am glad you like it. :)
